manifests/init.pp

```diff
@@ -151,6 +153,7 @@
       key     => $certs::apache::apache_key,
       ca_cert => $certs::ca_cert,
       require => Class['certs::apache'],
+      port    => ${crane_port},
```

Comment:
The `{}` are only necessary when the variable is in an interpolated string. You can just do `$crane_port`.